### PR TITLE
[wip] Fix yaml errors/warnings in all *.yml and *.yaml files

### LINKED ISSUE #8560: yamllint errors
##### Bug report summary

There is bunch of warning/errors reported by yamllint.
We need to tune config/fix errors.

___

non-python.d yaml files

```cmd
./.codeclimate.yml
  46:7      error    wrong indentation: expected 8 but found 6  (indentation)
  99:1      error    too many blank lines (1 > 0)  (empty-lines)

./web/api/netdata-swagger.yaml
  451:12    error    wrong indentation: expected 12 but found 11  (indentation)
  452:15    error    wrong indentation: expected 13 but found 14  (indentation)
  473:12    error    wrong indentation: expected 12 but found 11  (indentation)
  474:15    error    wrong indentation: expected 13 but found 14  (indentation)

./build_external/projects/aclk-testing/paho-compose.yml
  6:34      error    no new line character at the end of file  (new-line-at-end-of-file)

./build_external/projects/aclk-testing/vernemq-compose.yml
  7:1       error    too many blank lines (1 > 0)  (empty-lines)

./build_external/projects/aclk-testing/agent-valgrind-compose.yml
  8:7       error    wrong indentation: expected 8 but found 6  (indentation)
  18:5      error    wrong indentation: expected 6 but found 4  (indentation)
  19:1      error    too many blank lines (1 > 0)  (empty-lines)

./build_external/projects/aclk-testing/agent-compose.yml
  8:7       error    wrong indentation: expected 8 but found 6  (indentation)
  18:5      error    wrong indentation: expected 6 but found 4  (indentation)
  19:1      error    too many blank lines (1 > 0)  (empty-lines)

./build_external/projects/master-slaves/docker-compose.yml
  7:5       error    wrong indentation: expected 6 but found 4  (indentation)
  9:5       error    wrong indentation: expected 6 but found 4  (indentation)

./build_external/projects/only-agent/docker-compose.yml
  7:5       error    wrong indentation: expected 6 but found 4  (indentation)
```

___

python.d configs

```cmd
./collectors/python.d.plugin/freeradius/freeradius.conf
  66:51     error    trailing spaces  (trailing-spaces)
  74:70     error    trailing spaces  (trailing-spaces)
  79:45     error    trailing spaces  (trailing-spaces)

./collectors/python.d.plugin/sensors/sensors.conf
  61:1      error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/exim/exim.conf
  1:49      error    trailing spaces  (trailing-spaces)

./collectors/python.d.plugin/gearman/gearman.conf
  70:11     error    too many spaces before colon  (colons)
  71:11     error    too many spaces before colon  (colons)
  72:11     error    too many spaces before colon  (colons)
  72:18     error    no new line character at the end of file  (new-line-at-end-of-file)

./collectors/python.d.plugin/dovecot/dovecot.conf
  77:11     error    too many spaces before colon  (colons)
  78:11     error    too many spaces before colon  (colons)
  79:11     error    too many spaces before colon  (colons)
  82:11     error    too many spaces before colon  (colons)
  83:11     error    too many spaces before colon  (colons)
  84:11     error    too many spaces before colon  (colons)
  87:11     error    too many spaces before colon  (colons)
  88:11     error    too many spaces before colon  (colons)
  89:11     error    too many spaces before colon  (colons)
  92:11     error    too many spaces before colon  (colons)
  93:11     error    too many spaces before colon  (colons)
  96:11     error    too many spaces before colon  (colons)
  97:11     error    too many spaces before colon  (colons)
  98:1      error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/tomcat/tomcat.conf
  80:7      error    too many spaces before colon  (colons)
  81:7      error    too many spaces before colon  (colons)
  84:7      error    too many spaces before colon  (colons)
  85:7      error    too many spaces before colon  (colons)
  88:7      error    too many spaces before colon  (colons)
  89:7      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/ipfs/ipfs.conf
  75:9      error    too many spaces before colon  (colons)
  76:9      error    too many spaces before colon  (colons)
  77:9      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/redis/redis.conf
  79:11     error    too many spaces before colon  (colons)
  80:11     error    too many spaces before colon  (colons)
  84:11     error    too many spaces before colon  (colons)
  85:11     error    too many spaces before colon  (colons)
  89:11     error    too many spaces before colon  (colons)
  90:11     error    too many spaces before colon  (colons)
  94:11     error    too many spaces before colon  (colons)
  95:11     error    too many spaces before colon  (colons)
  96:11     error    too many spaces before colon  (colons)
  100:11    error    too many spaces before colon  (colons)
  101:11    error    too many spaces before colon  (colons)
  102:11    error    too many spaces before colon  (colons)
  106:11    error    too many spaces before colon  (colons)
  107:11    error    too many spaces before colon  (colons)
  108:11    error    too many spaces before colon  (colons)
  110:1     error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/dnsdist/dnsdist.conf
  83:1      error    too many blank lines (2 > 0)  (empty-lines)

./collectors/python.d.plugin/bind_rndc/bind_rndc.conf
  73:45     error    trailing spaces  (trailing-spaces)
  86:2      error    trailing spaces  (trailing-spaces)
  94:2      error    trailing spaces  (trailing-spaces)

./collectors/python.d.plugin/ovpn_status_log/ovpn_status_log.conf
  76:89     error    trailing spaces  (trailing-spaces)
  88:2      error    trailing spaces  (trailing-spaces)

./collectors/python.d.plugin/squid/squid.conf
  93:11     error    too many spaces before colon  (colons)
  94:11     error    too many spaces before colon  (colons)
  95:11     error    too many spaces before colon  (colons)
  96:11     error    too many spaces before colon  (colons)
  99:11     error    too many spaces before colon  (colons)
  100:11    error    too many spaces before colon  (colons)
  101:11    error    too many spaces before colon  (colons)
  102:11    error    too many spaces before colon  (colons)
  105:11    error    too many spaces before colon  (colons)
  106:11    error    too many spaces before colon  (colons)
  107:11    error    too many spaces before colon  (colons)
  108:11    error    too many spaces before colon  (colons)
  111:11    error    too many spaces before colon  (colons)
  112:11    error    too many spaces before colon  (colons)
  113:11    error    too many spaces before colon  (colons)
  114:11    error    too many spaces before colon  (colons)
  119:11    error    too many spaces before colon  (colons)
  120:11    error    too many spaces before colon  (colons)
  121:11    error    too many spaces before colon  (colons)
  122:11    error    too many spaces before colon  (colons)
  125:11    error    too many spaces before colon  (colons)
  126:11    error    too many spaces before colon  (colons)
  127:11    error    too many spaces before colon  (colons)
  128:11    error    too many spaces before colon  (colons)
  131:11    error    too many spaces before colon  (colons)
  132:11    error    too many spaces before colon  (colons)
  133:11    error    too many spaces before colon  (colons)
  134:11    error    too many spaces before colon  (colons)
  137:11    error    too many spaces before colon  (colons)
  138:11    error    too many spaces before colon  (colons)
  139:11    error    too many spaces before colon  (colons)
  140:11    error    too many spaces before colon  (colons)
  145:11    error    too many spaces before colon  (colons)
  146:11    error    too many spaces before colon  (colons)
  147:11    error    too many spaces before colon  (colons)
  148:11    error    too many spaces before colon  (colons)
  151:11    error    too many spaces before colon  (colons)
  152:11    error    too many spaces before colon  (colons)
  153:11    error    too many spaces before colon  (colons)
  154:11    error    too many spaces before colon  (colons)
  157:11    error    too many spaces before colon  (colons)
  158:11    error    too many spaces before colon  (colons)
  159:11    error    too many spaces before colon  (colons)
  160:11    error    too many spaces before colon  (colons)
  163:11    error    too many spaces before colon  (colons)
  164:11    error    too many spaces before colon  (colons)
  165:11    error    too many spaces before colon  (colons)
  166:11    error    too many spaces before colon  (colons)
  167:1     error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/httpcheck/httpcheck.conf
  107:1     error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/mongodb/mongodb.conf
  92:9      error    too many spaces before colon  (colons)
  93:9      error    too many spaces before colon  (colons)
  94:9      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/memcached/memcached.conf
  77:11     error    too many spaces before colon  (colons)
  78:11     error    too many spaces before colon  (colons)
  79:11     error    too many spaces before colon  (colons)
  82:11     error    too many spaces before colon  (colons)
  83:11     error    too many spaces before colon  (colons)
  84:11     error    too many spaces before colon  (colons)
  87:11     error    too many spaces before colon  (colons)
  88:11     error    too many spaces before colon  (colons)
  89:11     error    too many spaces before colon  (colons)
  90:1      error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/postgres/postgres.conf
  96:36     error    trailing spaces  (trailing-spaces)
  111:13    error    too many spaces before colon  (colons)
  112:13    error    too many spaces before colon  (colons)
  113:13    error    too many spaces before colon  (colons)
  116:13    error    too many spaces before colon  (colons)
  117:13    error    too many spaces before colon  (colons)
  118:13    error    too many spaces before colon  (colons)
  119:13    error    too many spaces before colon  (colons)
  120:13    error    too many spaces before colon  (colons)
  123:13    error    too many spaces before colon  (colons)
  124:13    error    too many spaces before colon  (colons)
  125:13    error    too many spaces before colon  (colons)
  126:13    error    too many spaces before colon  (colons)
  127:13    error    too many spaces before colon  (colons)
  130:13    error    too many spaces before colon  (colons)
  131:13    error    too many spaces before colon  (colons)
  132:13    error    too many spaces before colon  (colons)
  133:13    error    too many spaces before colon  (colons)
  134:13    error    too many spaces before colon  (colons)

./collectors/python.d.plugin/go_expvar/go_expvar.conf
  108:1     error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/samba/samba.conf
  60:71     error    no new line character at the end of file  (new-line-at-end-of-file)

./collectors/python.d.plugin/nginx/nginx.conf
  6:2       error    trailing spaces  (trailing-spaces)
  84:2      error    trailing spaces  (trailing-spaces)
  97:7      error    too many spaces before colon  (colons)
  98:7      error    too many spaces before colon  (colons)
  101:7     error    too many spaces before colon  (colons)
  102:7     error    too many spaces before colon  (colons)
  105:7     error    too many spaces before colon  (colons)
  106:7     error    too many spaces before colon  (colons)
  107:1     error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/nginx_plus/nginx_plus.conf
  76:7      error    too many spaces before colon  (colons)
  77:7      error    too many spaces before colon  (colons)
  80:7      error    too many spaces before colon  (colons)
  81:7      error    too many spaces before colon  (colons)
  84:7      error    too many spaces before colon  (colons)
  85:7      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/mysql/mysql.conf
  114:11    error    too many spaces before colon  (colons)
  115:11    error    too many spaces before colon  (colons)
  118:11    error    too many spaces before colon  (colons)
  119:11    error    too many spaces before colon  (colons)
  122:11    error    too many spaces before colon  (colons)
  123:11    error    too many spaces before colon  (colons)
  126:11    error    too many spaces before colon  (colons)
  129:11    error    too many spaces before colon  (colons)
  132:11    error    too many spaces before colon  (colons)
  135:11    error    too many spaces before colon  (colons)
  138:11    error    too many spaces before colon  (colons)
  141:11    error    too many spaces before colon  (colons)
  144:11    error    too many spaces before colon  (colons)
  147:11    error    too many spaces before colon  (colons)
  150:11    error    too many spaces before colon  (colons)
  153:11    error    too many spaces before colon  (colons)
  154:11    error    too many spaces before colon  (colons)
  159:11    error    too many spaces before colon  (colons)
  162:11    error    too many spaces before colon  (colons)
  163:11    error    too many spaces before colon  (colons)
  166:11    error    too many spaces before colon  (colons)
  169:11    error    too many spaces before colon  (colons)
  170:11    error    too many spaces before colon  (colons)
  178:11    error    too many spaces before colon  (colons)
  179:11    error    too many spaces before colon  (colons)
  180:11    error    too many spaces before colon  (colons)
  183:11    error    too many spaces before colon  (colons)
  184:11    error    too many spaces before colon  (colons)
  185:11    error    too many spaces before colon  (colons)
  188:11    error    too many spaces before colon  (colons)
  189:11    error    too many spaces before colon  (colons)
  191:11    error    too many spaces before colon  (colons)
  194:11    error    too many spaces before colon  (colons)
  195:11    error    too many spaces before colon  (colons)
  197:11    error    too many spaces before colon  (colons)
  200:11    error    too many spaces before colon  (colons)
  201:11    error    too many spaces before colon  (colons)
  203:11    error    too many spaces before colon  (colons)
  206:11    error    too many spaces before colon  (colons)
  207:11    error    too many spaces before colon  (colons)
  209:11    error    too many spaces before colon  (colons)
  212:11    error    too many spaces before colon  (colons)
  213:11    error    too many spaces before colon  (colons)
  215:11    error    too many spaces before colon  (colons)
  216:11    error    too many spaces before colon  (colons)
  221:11    error    too many spaces before colon  (colons)
  222:11    error    too many spaces before colon  (colons)
  224:11    error    too many spaces before colon  (colons)
  225:11    error    too many spaces before colon  (colons)
  228:11    error    too many spaces before colon  (colons)
  229:11    error    too many spaces before colon  (colons)
  231:11    error    too many spaces before colon  (colons)
  232:11    error    too many spaces before colon  (colons)
  238:11    error    too many spaces before colon  (colons)
  239:11    error    too many spaces before colon  (colons)
  240:11    error    too many spaces before colon  (colons)
  243:11    error    too many spaces before colon  (colons)
  244:11    error    too many spaces before colon  (colons)
  245:11    error    too many spaces before colon  (colons)
  248:11    error    too many spaces before colon  (colons)
  249:11    error    too many spaces before colon  (colons)
  251:11    error    too many spaces before colon  (colons)
  254:11    error    too many spaces before colon  (colons)
  255:11    error    too many spaces before colon  (colons)
  257:11    error    too many spaces before colon  (colons)
  260:11    error    too many spaces before colon  (colons)
  261:11    error    too many spaces before colon  (colons)
  263:11    error    too many spaces before colon  (colons)
  266:11    error    too many spaces before colon  (colons)
  267:11    error    too many spaces before colon  (colons)
  269:11    error    too many spaces before colon  (colons)
  272:11    error    too many spaces before colon  (colons)
  273:11    error    too many spaces before colon  (colons)
  275:11    error    too many spaces before colon  (colons)
  276:11    error    too many spaces before colon  (colons)
  281:11    error    too many spaces before colon  (colons)
  282:11    error    too many spaces before colon  (colons)
  284:11    error    too many spaces before colon  (colons)
  285:11    error    too many spaces before colon  (colons)
  288:11    error    too many spaces before colon  (colons)
  289:11    error    too many spaces before colon  (colons)
  291:11    error    too many spaces before colon  (colons)
  292:11    error    too many spaces before colon  (colons)
  293:1     error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/chrony/chrony.conf
  1:51      error    trailing spaces  (trailing-spaces)

./collectors/python.d.plugin/monit/monit.conf
  85:7      error    too many spaces before colon  (colons)
  86:7      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/icecast/icecast.conf
  76:7      error    too many spaces before colon  (colons)
  77:7      error    too many spaces before colon  (colons)
  80:7      error    too many spaces before colon  (colons)
  81:7      error    too many spaces before colon  (colons)
  81:49     error    no new line character at the end of file  (new-line-at-end-of-file)

./collectors/python.d.plugin/portcheck/portcheck.conf
  74:1      error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/phpfpm/phpfpm.conf
  78:7      error    too many spaces before colon  (colons)
  79:7      error    too many spaces before colon  (colons)
  82:7      error    too many spaces before colon  (colons)
  83:7      error    too many spaces before colon  (colons)
  86:7      error    too many spaces before colon  (colons)
  87:7      error    too many spaces before colon  (colons)
  88:1      error    too many blank lines (1 > 0)  (empty-lines)

./collectors/python.d.plugin/uwsgi/uwsgi.conf
  76:11     error    too many spaces before colon  (colons)
  77:11     error    too many spaces before colon  (colons)
  80:11     error    too many spaces before colon  (colons)
  81:11     error    too many spaces before colon  (colons)
  82:11     error    too many spaces before colon  (colons)
  85:11     error    too many spaces before colon  (colons)
  86:11     error    too many spaces before colon  (colons)
  87:11     error    too many spaces before colon  (colons)
  90:11     error    too many spaces before colon  (colons)
  91:11     error    too many spaces before colon  (colons)
  92:11     error    too many spaces before colon  (colons)

./collectors/python.d.plugin/riakkv/riakkv.conf
  68:7      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/proxysql/proxysql.conf
  77:11     error    too many spaces before colon  (colons)
  78:11     error    too many spaces before colon  (colons)
  79:11     error    too many spaces before colon  (colons)
  80:11     error    too many spaces before colon  (colons)
  81:11     error    too many spaces before colon  (colons)
  84:11     error    too many spaces before colon  (colons)
  85:11     error    too many spaces before colon  (colons)
  86:11     error    too many spaces before colon  (colons)
  87:11     error    too many spaces before colon  (colons)
  88:11     error    too many spaces before colon  (colons)
  91:11     error    too many spaces before colon  (colons)
  92:11     error    too many spaces before colon  (colons)
  93:11     error    too many spaces before colon  (colons)
  94:11     error    too many spaces before colon  (colons)
  95:11     error    too many spaces before colon  (colons)
  98:11     error    too many spaces before colon  (colons)
  99:11     error    too many spaces before colon  (colons)
  100:11    error    too many spaces before colon  (colons)
  101:11    error    too many spaces before colon  (colons)
  102:11    error    too many spaces before colon  (colons)
  105:11    error    too many spaces before colon  (colons)
  106:11    error    too many spaces before colon  (colons)
  107:11    error    too many spaces before colon  (colons)
  108:11    error    too many spaces before colon  (colons)
  109:11    error    too many spaces before colon  (colons)
  112:11    error    too many spaces before colon  (colons)
  113:11    error    too many spaces before colon  (colons)
  114:11    error    too many spaces before colon  (colons)
  115:11    error    too many spaces before colon  (colons)
  116:11    error    too many spaces before colon  (colons)

./collectors/python.d.plugin/litespeed/litespeed.conf
  71:7      error    too many spaces before colon  (colons)
  72:8      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/oracledb/oracledb.conf
  84:17     error    no new line character at the end of file  (new-line-at-end-of-file)

./collectors/python.d.plugin/apache/apache.conf
  76:7      error    too many spaces before colon  (colons)
  77:7      error    too many spaces before colon  (colons)
  80:7      error    too many spaces before colon  (colons)
  81:7      error    too many spaces before colon  (colons)
  84:7      error    too many spaces before colon  (colons)
  85:7      error    too many spaces before colon  (colons)

./collectors/python.d.plugin/dns_query_time/dns_query_time.conf
  69:73     error    no new line character at the end of file  (new-line-at-end-of-file)
```

___

##### OS / Environment

```
any
```

##### Netdata version

 
`netdata v1.20.0-286-g8fe7485a`

##### Component Name

- ci
- external/python

##### Steps To Reproduce

1. `cd <netdata source code>`
2. `docker run --rm -v $(pwd):/data cytopia/yamllint -f colored --no-warnings -c .yamllint.yml .`

##### Expected behavior

`yamllint` reports no errors

### PR DESCRIPTION
**Pull request by prologic**
Fixes #8560

##### Summary

ssia

##### Component Name

- code quality / lint
- area/ci

##### Test Plan

```#!sh
$ yamllint $(find . -type f \( -name '*.yaml' -o -name '*.yml' \) | paste -s -d' ') | wc -l
0
```

##### Additional Information

### CODE REVIEW

**Comment by prologic** [comment]
@ilyam8 Do you want to fix all the python.d configs? :)

**Comment by ilyam8** [comment]
ill take care of them 😉

**Comment by prologic** [comment]
> ill take care of them 😉

Do you just want to add your commits to this PR?

**Comment by ilyam8** [comment]
> Do you just want to add your commits to this PR?

Yes.

**Comment by thiagoftsm** [comment]
@prologic when I ran on my environment the test plan I got this:

```
$ git status
On branch fix_yaml_errors
nothing to commit, working tree clean
$ cat .squash.yml
---
deployments:
  netdata:
    auto_deploy_on_commits: true
    filename: ./packaging/docker/Dockerfile
    context_path: ./
    port_forwarding: 80:19999
    run_options: >
      -v /proc:/host/proc:ro
      -v /sys:/host/sys:ro
      -v /var/run/docker.sock:/var/run/docker.sock:ro
      --cap-add SYS_PTRACE
      --security-opt apparmor=unconfined
$ /root/.local/bin/yamllint --version
yamllint 1.23.0
```

**Comment by ilyam8** [comment]
@thiagoftsm this PR is wip, dont test it

File: .travis.yml [hunk 1]
```diff
@@ -197,10 +263,13 @@ jobs:
         git:
           depth: false
         before_install:
-          - sudo apt-get install -y wget lxc python3-lxc python-lxc lxc-templates dh-make git-buildpackage build-essential libdistro-info-perl
+          - sudo apt-get install -y wget lxc python3-lxc python-lxc \
+            lxc-templates dh-make git-buildpackage build-essential \
+            libdistro-info-perl
```
Comment (knatsakis) [inline .travis.yml:268]: I think the backslashes are not needed here

File: .travis.yml [hunk 2]
```diff
@@ -529,4 +620,7 @@ jobs:
     - stage: Trigger deb and rpm package build (nightly release)
       name: Trigger deb and rpm package build
       script: .travis/trigger_package_generation.sh "[Build latest]"
-      after_failure: post_message "TRAVIS_MESSAGE" "<!here> Failed to trigger deb and rpm package build during nightly release" "${NOTIF_CHANNEL}"
+      after_failure: >
+        post_message "TRAVIS_MESSAGE"
+        "<!here> Failed to trigger deb and rpm package build during nightly release"
+        "${NOTIF_CHANNEL}"
```
Comment (knatsakis) [inline .travis.yml:626]:
```
diff -urN <(cat .travis.yml | yq .) <(git show fb072738d387dd7623b6f995a86b694f8cca37ac:.travis.yml | yq .)
```
:point_up: indicates that the before and after `.travis.yml`s are not equivalent

**Comment by prologic** [comment]
@knatsakis You are right; the difference I see is an extra `]n` you get (_but this id well documented_):

```#!sh
(⎈ |netdata-testing:infra)
prologic@Jamess-iMac
Mon May 25 20:51:44
~/tmp
 0
$ yj -y < foo.yml  | jq '.'
{
  "foo": "This is a very long string over multiple lines\n"
}
(⎈ |netdata-testing:infra)
prologic@Jamess-iMac
Mon May 25 20:51:47
~/tmp
 0 0
$ yj -y < bar.yml  | jq '.'
{
  "foo": "This is a very long string over multiple lines"
}
(⎈ |netdata-testing:infra)
prologic@Jamess-iMac
Mon May 25 20:51:50
~/tmp
 0 0
$ cat foo.yml
---
foo: >
  This is a
  very long string
  over multiple lines
(⎈ |netdata-testing:infra)
prologic@Jamess-iMac
Mon May 25 20:52:12
~/tmp
 0
$ cat bar.yml
---
foo: This is a very long string over multiple lines
```

**Comment by prologic** [comment]
I don't have time to dig into this. Experience has taught me time and again that trying to do en-mass without sufficient tooling and safeguards is rather hard to do. We lack both the tooling and safeguards to do this properly en-mass. We should instead just do this piece-meal as/when it comes up in other work we do and throughly test _those_ changes in smaller chunks.

**Pull request closed without merging**